src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -150,13 +150,13 @@ public DurableTaskExtension()
             this.nameResolver = nameResolver ?? throw new ArgumentNullException(nameof(nameResolver));
             this.loggerFactory = loggerFactory ?? throw new ArgumentNullException(nameof(loggerFactory));
             this.PlatformInformationService = platformInformationService ?? throw new ArgumentNullException(nameof(platformInformationService));
-            this.ResolveAppSettingOptions();
+            DurableTaskOptions.ResolveAppSettingOptions(this.Options, this.nameResolver);
 

```

Comment:
made this method static so it may be re-used in `DurableTaskTriggersScaleProvider`